header.php

```diff
@@ -27,6 +27,34 @@
 
 <section id="main-container">
 
-  <header id="header">
-    <h1><a href="<?php echo home_url(); ?>"><?php bloginfo('name'); ?></a></h1>
-  </header>
\ No newline at end of file
+<header id="header" class="container">
+  <h1 class="u-visuallyhidden"><?php bloginfo('name'); ?></h1>
+  <nav class="text-align-center">
+    <ul class="u-inline-list">
+      <li><a href="<?php echo home_url(); ?>">WORK</a></li>
```

Comment:
haha "calm". 😆 i usually do but then i was thinking what does that do to copy paste? like how would we want that to be for screenreaders etc...I guess the text itself should just be capitalized like "Work"? yea